assets/src/dashboard/components/cardGridItem/card-preview.js

```diff
@@ -126,11 +124,11 @@ const CardPreviewContainer = ({
               </PreviewButton>
             </PreviewContainer>
           )}
-          {onOpenInEditorClick && (
+          {editUrl && (
```

Comment:
nit: if editUrl is always required now then we shouldn't need to check on it existing here and just always render the `CtaContainer`.